src/mon/MonCommands.h

```diff
@@ -645,7 +645,7 @@ COMMAND("osd crush tree "
         "name=shadow,type=CephChoices,strings=--show-shadow,req=false", \
 	"dump crush buckets and items in a tree view",
 	"osd", "r", "cli,rest")
-COMMAND("osd crush ls name=node,type=CephString,goodchars=goodchars=[A-Za-z0-9-_.]",
+COMMAND("osd crush ls name=node,type=CephString,goodchars=[A-Za-z0-9-_.]",
```

Comment:
could you add a test in `qa/standalone/mon/osd-crush.sh`? also,
following command completes successfully in my testbed:
```sh
ceph osd crush ls osd.0
```
>  that result in chars check failure

could you share the steps that leads you to the check failure?

Reply:
added testcase for `osd crush ls` .
I mean that the following command get ENOENT error
```
[root@ceph35 ~]# ceph osd crush ls ceph37=[-@
Error ENOENT: node 'ceph37=[-@' does not exist
```
while we except an EINVAL error due to invalid chars like below:
```
[root@ceph35 ~]# ceph osd crush class ls-osd sdd=[-@
Invalid command:  invalid chars @[= in sdd=[-@
osd crush class ls-osd <class> :  list all osds belonging to the specific <class>
Error EINVAL: invalid command
```
Actually, it will not lead to check failure for normal use, so I change the commit title, sorry for the ambiguity